<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1843</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>1766</width>
      <height>731</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1766</width>
      <height>0</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://c8.alamy.com/compes/hg634c/mapa-de-carreteras-de-honduras-hg634c.jpg&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Mapa de Honduras&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Rutas</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Origen:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Origen">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Destino:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRutas">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Ver Rutas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tipo de Transporte:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_transporte">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Destino">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Lb_KmRestante">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIniciarSimulacion">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>380</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Iniciar Viaje</string>
      </property>
     </widget>
     <widget class="QLabel" name="Lb_TiempoRestante">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="Lb_HoraLlegada">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QListWidget" name="List_rutas">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_seleccionRuta">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleccionar Ruta</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1843</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
